Nil pointer during cluster creation

Version: v1.0.1

Log:
```
time="2020-11-26T08:39:15Z" level=info msg="Start creating cluster"
time="2020-11-26T08:39:15Z" level=info msg="Start setting state from provided opts: \nbool_options:<key:\"bmsAutoRenew\" value:false > string_options:<key:\"accessKey\" value:\"\" > string_options:<key:\"authProxyCa\" value:\"\" > string_options:<key:\"authenticationMode\" value:\"rbac\" > string_options:<key:\"availabilityZone\" value:\"eu-de-01\" > string_options:<key:\"bmsPeriodType\" value:\"month\" > string_options:<key:\"clusterEipShareType\" value:\"PER\" > string_options:<key:\"clusterEipType\" value:\"5_bgp\" > string_options:<key:\"clusterFlavor\" value:\"cce.s1.medium\" > string_options:<key:\"clusterFloatingIp\" value:\"\" > string_options:<key:\"clusterType\" value:\"VirtualMachine\" > string_options:<key:\"clusterVersion\" value:\"\" > string_options:<key:\"containerNetworkCidr\" value:\"172.16.0.0/16\" > string_options:<key:\"containerNetworkMode\" value:\"overlay_l2\" > string_options:<key:\"dataVolumeType\" value:\"SATA\" > string_options:<key:\"description\" value:\"\" > string_options:<key:\"displayName\" value:\"sandbox\" > string_options:<key:\"domainName\" value:\"OTC00000000001000000448\" > string_options:<key:\"driverName\" value:\"kd-lm7gf\" > string_options:<key:\"highwaySubnet\" value:\"\" > string_options:<key:\"keyPair\" value:\"KeyPair-4f26\" > string_options:<key:\"loadBalancer\" value:\"\" > string_options:<key:\"name\" value:\"c-4vrjc\" > string_options:<key:\"nodeFlavor\" value:\"s2.xlarge.1\" > string_options:<key:\"nodeOs\" value:\"EulerOS 2.5\" > string_options:<key:\"password\" value:\"***\" > string_options:<key:\"projectName\" value:\"eu-de_test_dmd\" > string_options:<key:\"region\" value:\"eu-de\" > string_options:<key:\"rootVolumeType\" value:\"SATA\" > string_options:<key:\"secretKey\" value:\"\" > string_options:<key:\"subnet\" value:\"\" > string_options:<key:\"subnetId\" value:\"0f9d5426-ab11-471f-b5a0-d7b413cd2cfa\" > string_options:<key:\"token\" value:\"\" > string_options:<key:\"username\" value:\"akachurin\" > string_options:<key:\"vpc\" value:\"\" > string_options:<key:\"vpcId\" value:\"a82d8c31-1f1b-4d55-bd45-e01ec3de417c\" > int_options:<key:\"billingMode\" value:0 > int_options:<key:\"bmsPeriodNum\" value:1 > int_options:<key:\"clusterBillingMode\" value:0 > int_options:<key:\"clusterEipBandwidthSize\" value:100 > int_options:<key:\"dataVolumeSize\" value:100 > int_options:<key:\"nodeCount\" value:2 > int_options:<key:\"rootVolumeSize\" value:40 > string_slice_options:<key:\"clusterLabels\" value:<value:\"origin=rancher-otc\" > > "
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x139ea03]

goroutine 19 [running]:
github.com/opentelekomcloud/kontainer-engine-driver-otc/opentelekomcloud.(*CCEDriver).Create(0xc0004208a0, 0x18818c0, 0xc000420de0, 0xc00040a700, 0x0, 0x0, 0x0, 0x0)
	github.com/opentelekomcloud/kontainer-engine-driver-otc/opentelekomcloud/driver.go:639 +0x243
github.com/rancher/kontainer-engine/types.(*GrpcServer).Create(0xc0001bde60, 0x18818c0, 0xc000420de0, 0xc000420e10, 0xc0001bde60, 0xc000420de0, 0xc00010bb78)
	github.com/rancher/kontainer-engine@v0.0.4-dev.0.20200406202044-bf3f55d3710a/types/rpc_server.go:42 +0x79
github.com/rancher/kontainer-engine/types._Driver_Create_Handler(0x163e920, 0xc0001bde60, 0x18818c0, 0xc000420de0, 0xc0003c44e0, 0x0, 0x18818c0, 0xc000420de0, 0xc000136480, 0x43c)
	github.com/rancher/kontainer-engine@v0.0.4-dev.0.20200406202044-bf3f55d3710a/types/drivers.pb.go:1436 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00042e600, 0x1890ba0, 0xc00042e900, 0xc0000b2000, 0xc000420a80, 0x2211060, 0x0, 0x0, 0x0)
	google.golang.org/grpc@v1.26.0/server.go:1024 +0x522
google.golang.org/grpc.(*Server).handleStream(0xc00042e600, 0x1890ba0, 0xc00042e900, 0xc0000b2000, 0x0)
	google.golang.org/grpc@v1.26.0/server.go:1313 +0xd34
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc0001fcab0, 0xc00042e600, 0x1890ba0, 0xc00042e900, 0xc0000b2000)
	google.golang.org/grpc@v1.26.0/server.go:722 +0xa5
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.26.0/server.go:720 +0xa5
2020/11/26 08:39:15 [INFO] kontainerdriver kd-lm7gf stopped
2020/11/26 08:39:15 [ERROR] ClusterController c-4vrjc [cluster-provisioner-controller] failed with : rpc error: code = Unavailable desc = transport is closing
```

--- a/opentelekomcloud/driver.go
+++ b/opentelekomcloud/driver.go
@@ -626,6 +626,10 @@ func cleanupManagedResources(client services.Client, state *clusterState) error 
 
 func (d *CCEDriver) Create(_ context.Context, opts *types.DriverOptions, info *types.ClusterInfo) (*types.ClusterInfo, error) {
 	logrus.Info("Start creating cluster")
+	if info == nil {
+		logrus.Debug("Info is nil, initialize info")
+		info = &types.ClusterInfo{}
+	}
 	logrus.Debug("Get state from opts")
 	state, err := optsToState(opts)
 	if err != nil {

--- a/opentelekomcloud/driver_test.go
+++ b/opentelekomcloud/driver_test.go
@@ -167,7 +167,7 @@ func TestDriver_ClusterWorkflow(t *testing.T) {
 	}()
 
 	driver := NewDriver()
-	info, err := driver.Create(ctx, driverOptions, &types.ClusterInfo{})
+	info, err := driver.Create(ctx, driverOptions, nil)
 	require.NoError(t, err)
 	t.Log("Cluster created")
 

